detections/network/detect_ipv6_network_infrastructure_threats.yml

```diff
@@ -0,0 +1,45 @@
+name: Detect IPv6 Network Infrastructure Threats
+id: c3be767e-7959-44c5-8976-0e9c12a91ad2
+version: 1
+date: '2020-10-28'
+description: By enabling IPv6 First Hop Security as a Layer 2 Security measure on the organization's 
+  network devices, we will be able to detect various attacks such as packet forging in the Infrastructure.
+how_to_implement: This search uses a standard SPL query on logs from Cisco Network
+  devices. The network devices must be configured with one or more First Hop Security measures such as
+  RA Guard, DHCP Guard and/or device tracking. See References for more information.
+  The search also requires that the Cisco Networks Add-on for Splunk 
+  (https://splunkbase.splunk.com/app/1467) is used to
+  parse the logs from the Cisco network devices.
+type: ESCU
+references: ['https://www.ciscolive.com/c/dam/r/ciscolive/emea/docs/2019/pdf/BRKSEC-3200.pdf', 'https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/ipv6_fhsec/configuration/xe-16-12/ip6f-xe-16-12-book/ip6-ra-guard.html', 'https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/ipv6_fhsec/configuration/xe-16-12/ip6f-xe-16-12-book/ip6-snooping.html', 'https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/ipv6_fhsec/configuration/xe-16-12/ip6f-xe-16-12-book/ip6-dad-proxy.html', 'https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/ipv6_fhsec/configuration/xe-16-12/ip6f-xe-16-12-book/ip6-nd-mcast-supp.html', 'https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/ipv6_fhsec/configuration/xe-16-12/ip6f-xe-16-12-book/ip6-dhcpv6-guard.html', 'https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/ipv6_fhsec/configuration/xe-16-12/ip6f-xe-16-12-book/ip6-src-guard.html', 'https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/ipv6_fhsec/configuration/xe-16-12/ip6f-xe-16-12-book/ipv6-dest-guard.html']
+author: Mikael Bjerkeland, Splunk
+search: '`cisco_networks` (facility="SISF") | eval src_interface=src_int_prefix_long+src_int_suffix 
```

Comment:
@inspired these detections looks great for some low hanging fruit on the MITRE network matrix!\

I have just been testing the searches on a local instance only thing I am not sure about syntax wise is the value in some of the inital search parameters here (facility="SISF") is this ok?  

Thanks again super minor comment so far everything looks good

Reply:
@jzsplunk The SISF examples were taken from https://www.ciscolive.com/c/dam/r/ciscolive/emea/docs/2019/pdf/BRKSEC-3200.pdf

But it would actually make a lot of sense to also add the following to avoid false positives as there are other SISF events. Replace:

`(facility="SISF")`

with
`facility="SISF" mnemonic IN ("IP_THEFT","MAC_THEFT","MAC_AND_IP_THEFT","PAK_DROP")`

Want me to submit a new PR or will you change it?